strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/cron/jobs/RebuildMavenIndexesCronJob.java

```diff
@@ -27,52 +24,42 @@
 
```

Comment:
Clarification: Similar to my question for the other cron task, will this be created per repository automatically, or...?

Reply:
Yes, `RebuildMavenIndexesCronJob` is for hosted repository